plugins/src/main/java/com/google/fhir/gateway/plugin/ListAccessChecker.java

```diff
@@ -227,6 +228,22 @@ private AccessDecision processPatch(RequestDetailsReader requestDetails) throws
     return checkNonPatientAccessInUpdate(requestDetails, RequestTypeEnum.PATCH);
   }
 
+  private AccessDecision processDelete(RequestDetailsReader requestDetails) {
+    // We don't support deletion of List resource used as an access list for a user.
+    if (FhirUtil.isSameResourceType(requestDetails.getResourceName(), ResourceType.List)) {
+      if (patientListId.equals(FhirUtil.getIdOrNull(requestDetails))) {
```

Comment:
Hmm, shouldn't this be _not_ `equals`?

Reply:
Thanks for catching this. Intent was this  only but made a mistake below.
Fixed this and also added the required test case.